src/wmtk/simplex/cofaces_single_dimension.cpp

```diff
@@ -18,10 +18,15 @@ std::vector<Tuple> cofaces_single_dimension_tuples(
 {
     switch (mesh.top_simplex_type()) {
     case PrimitiveType::Face:
-        return cofaces_single_dimension_tuples(static_cast<const TriMesh&>(mesh), simplex, cofaces_type);
+        return cofaces_single_dimension_tuples(
```

Comment:
Just to make sure, you do use the clang-format file of this project right? I saw several formatting changes and there is a chance that they were not correctly formatted before but please make sure that it is not you who breaks the formatting.